protoc-gen-go/generator/generator.go

```diff
@@ -2508,6 +2509,97 @@ func (g *Generator) generateMessage(message *Descriptor) {
 	g.addInitf("%s.RegisterType((*%s)(nil), %q)", g.Pkg["proto"], ccTypeName, fullName)
 }
 
+func unescape(s string) string {
+	// NB: Sadly, we can't use strconv.Unquote because protoc will escape both
+	// single and double quotes, but strconv.Unquote only allows one or the
+	// other (based on actual surrounding quotes of its input argument).
+	s, err := doUnescape(s)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
`log.Fatal(err)`

Reply:
I just pushed a change so that it won't fail on bad input and simply propagate malformed escape sequences into the resulting bytes. WDYT?